reference/5.1/Microsoft.PowerShell.Core/Get-Help.md

```diff
@@ -98,16 +97,33 @@ articles that include that text in their contents.
 help files in the locale set for Windows, then in the parent locale, such as **pt** for **pt-BR**,
 and then in a fallback locale. Beginning in PowerShell 3.0, if `Get-Help` doesn't find help in the
 fallback locale, it looks for help articles in English, **en-US**, before it returns an error
-message or displaying auto-generated help.
+message or displaying autogenerated help.
 
-For information about the symbols that `Get-Help` displays in the command syntax diagram, see [about_Command_Syntax](./About/about_Command_Syntax.md).
-For information about parameter attributes, such as **Required** and **Position**, see [about_Parameters](./About/about_Parameters.md).
+For information about the symbols that `Get-Help` displays in the command syntax diagram, see
+[about_Command_Syntax](./About/about_Command_Syntax.md). For information about parameter attributes,
+such as **Required** and **Position**, see [about_Parameters](./About/about_Parameters.md).
 
->[!NOTE]
+> [!NOTE]
 > In PowerShell 3.0 and PowerShell 4.0, `Get-Help` can't find **About** articles in modules unless
-> the module is imported into the current session. This is a known issue. To get **About** articles
-> in a module, import the module, either by using the `Import-Module` cmdlet or by running a cmdlet
-> that's included in the module.
+> the module is imported into the current session. To get **About** articles in a module, import the
+> module using the `Import-Module` cmdlet or by running a cmdlet that's included in the module.
+
+Beginning with **PSReadLine** v2.2.2, the module ships with two functions that provide quick access
+to help while you are typing a command on the command line. The help is displayed in the terminal in
+an alternate screen buffer with paging.
+
+When you hit <kbd>F1</kbd> key, the **PSReadLine** `ShowCommandHelp` function invokes
+`Get-Help -Full` for the cmdlet name closest to the left of the cursor. When the cursor is
+immediately to the left of a parameter, the function jumps to that parameter's description in the
+full help topic. When you hit <kbd>Q</kbd> to exit the help view, you are returned to the command
+line at the same cursor position so you can continue typing the command.
+
+When you hit <kbd>Alt</kbd>-<kbd>h</kbd> key, the **PSReadLine** `ShowParameterHelp` function
```

Comment:
I'm not sure about this one either. We might want some standard language for our docs on how we talk about using key chords like this.

Maybe something like:

```suggestion
When you use the key combination <kbd>Alt</kbd>+<kbd>h</kbd>, the **PSReadLine** `ShowParameterHelp` function
```

I think we also use `<FirstKey>+<SecondKey>` typically rather than `<FirstKey>-<SecondKey>`.

Reply:
I did a search on `-` vs. `+` for our kbd markup. We are not consistent but `+` is the majority. I will fix.